tests/meson.build

```
@@ -14,6 +14,7 @@ library('nnscustom_drop_buffer',
 )
 
 # Build and copy exe for ssat
+copy = find_program('cp')
```

If this is mandatory to build the test, 
```suggestion
copy = find_program('cp', required: true)
```

---

Looks good.